--- libevent-2.1.8-stable/event.c.orig	2017-03-29 21:29:06.089923271 +0200
+++ libevent-2.1.8-stable/event.c	2017-03-29 21:31:33.269921590 +0200
@@ -881,10 +881,6 @@
 	if (base->evsel != NULL && base->evsel->dealloc != NULL)
 		base->evsel->dealloc(base);
 
-	for (i = 0; i < base->nactivequeues; ++i)
-		EVUTIL_ASSERT(TAILQ_EMPTY(&base->activequeues[i]));
-
-	EVUTIL_ASSERT(min_heap_empty_(&base->timeheap));
 	min_heap_dtor_(&base->timeheap);
 
 	mm_free(base->activequeues);
