--- libevent-2.0.3-alpha/event.c~	2009-11-19 17:43:22.000000000 +0000
+++ libevent-2.0.3-alpha/event.c	2009-11-24 15:02:31.550484522 +0000
@@ -460,16 +460,10 @@
 	if (base->evsel != NULL && base->evsel->dealloc != NULL)
 		base->evsel->dealloc(base);
 
-	for (i = 0; i < base->nactivequeues; ++i)
-		EVUTIL_ASSERT(TAILQ_EMPTY(&base->activequeues[i]));
-
-	EVUTIL_ASSERT(min_heap_empty(&base->timeheap));
 	min_heap_dtor(&base->timeheap);
 
 	mm_free(base->activequeues);
 
-	EVUTIL_ASSERT(TAILQ_EMPTY(&base->eventqueue));
-
 	evmap_io_clear(&base->io);
 	evmap_signal_clear(&base->sigmap);
 
