--- libevent-2.0.10-stable/Makefile.am~	2010-12-16 18:38:34.000000000 +0100
+++ libevent-2.0.10-stable/Makefile.am	2011-02-24 21:22:21.259717568 +0100
@@ -185,16 +185,17 @@
 
 if PTHREADS
 libevent_pthreads_la_SOURCES = evthread_pthread.c
+libevent_pthreads_la_LIBADD = libevent.la
 libevent_pthreads_la_LDFLAGS = $(GENERIC_LDFLAGS)
 endif
 
 libevent_extra_la_SOURCES = $(EXTRA_SRC)
-libevent_extra_la_LIBADD = $(MAYBE_CORE) $(SYS_LIBS)
+libevent_extra_la_LIBADD = $(MAYBE_CORE) $(SYS_LIBS) libevent.la
 libevent_extra_la_LDFLAGS = $(GENERIC_LDFLAGS)
 
 if OPENSSL
 libevent_openssl_la_SOURCES = bufferevent_openssl.c
-libevent_openssl_la_LIBADD = $(MAYBE_CORE) -lcrypto -lssl
+libevent_openssl_la_LIBADD = $(MAYBE_CORE) libevent.la -lcrypto -lssl
 libevent_openssl_la_LDFLAGS = $(GENERIC_LDFLAGS)
 endif
 
