--- libevent-2.0.12-stable/Makefile.am~	2011-06-03 21:45:36.000000000 +0200
+++ libevent-2.0.12-stable/Makefile.am	2011-06-07 09:08:30.650818312 +0200
@@ -193,17 +193,17 @@
 
 if PTHREADS
 libevent_pthreads_la_SOURCES = evthread_pthread.c
-libevent_pthreads_la_LIBADD = $(MAYBE_CORE)
+libevent_pthreads_la_LIBADD = $(MAYBE_CORE) libevent.la
 libevent_pthreads_la_LDFLAGS = $(GENERIC_LDFLAGS)
 endif
 
 libevent_extra_la_SOURCES = $(EXTRA_SRC)
-libevent_extra_la_LIBADD = $(MAYBE_CORE) $(SYS_LIBS)
+libevent_extra_la_LIBADD = $(MAYBE_CORE) $(SYS_LIBS) libevent.la
 libevent_extra_la_LDFLAGS = $(GENERIC_LDFLAGS)
 
 if OPENSSL
 libevent_openssl_la_SOURCES = bufferevent_openssl.c
-libevent_openssl_la_LIBADD = $(MAYBE_CORE) $(OPENSSL_LIBS)
+libevent_openssl_la_LIBADD = $(MAYBE_CORE) $(OPENSSL_LIBS) libevent.la
 libevent_openssl_la_LDFLAGS = $(GENERIC_LDFLAGS)
 endif
 
